src/components/Contacts/ContactListTable.jsx

```diff
@@ -44,31 +50,74 @@ const ContactListTable = ({ contacts, deleteContact }) => {
   const contactsCopy = [...contacts];
   const sortedContacts = contactsCopy.sort(comparePerson);
 
+  const columnTitlesArray = [
+    {
+      field: 'First Name',
+      width: 120,
+      headerAlign: 'center',
+      align: 'center'
+    },
+    {
+      field: 'Last Name',
+      width: 120,
+      headerAlign: 'center',
+      align: 'center'
+    },
+    {
+      field: 'Profile',
+      renderCell: (contactData) => <ContactProfileIcon contact={contactData} />,
+      sortable: false,
+      filterable: false,
+      width: 70,
+      headerAlign: 'center',
+      align: 'center'
+    },
+    {
+      field: 'actions',
+      type: 'actions',
+      width: 70,
+      getActions: (contactData) => [
```

Comment:
Although it's not essential, I think it'd be nicer to have a title for the delete icons. If only because it's a bit odd to have one column without a title. It's currently labeled in the documents table.

Adding `headerName: 'Actions',` here would accomplish this.

Reply:
I'll add one. I'll expand the width of this too.

Reply:
Got this included.